gen/Cargo.toml

```
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-pact-data-model = { git = "https://github.com/sine-fdn/pact-data-model.git", branch = "extensions-type" }
+pact-data-model = { git = "https://github.com/sine-fdn/pact-data-model.git", commit = "73ba24f6aef83eeb933b730d3a0c1d90405906bd" }
```

Coul we perhaps import it like so?
```suggestion
pact-data-model = { git = "https://github.com/sine-fdn/pact-data-model.git" }
```